addons/UtilityClasses/ImageProcessingLibrary.cpp

```diff
@@ -166,7 +166,7 @@ status_t ImageProcessingLibrary::gaussian_blur(BBitmap *bitmap,float radius, int
 	int32 d_bpr = intermediate->BytesPerRow()/4;
 
 	// Start the threads.
-	thread_id blur_thread_array[B_MAX_CPU_COUNT];
+	thread_id blur_thread_array[8];
```

Comment:
Might want to add a TODO about deciding how many threads to use based on CPU count.